src/Symfony/Component/Routing/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -121,8 +130,8 @@ public function testLoadRouteWithControllerAttribute()
 
     public function testLoadRouteWithoutControllerAttribute()
     {
-        $loader = new YamlFileLoader(new FileLocator(array(__DIR__.'/../Fixtures/controller')));
-        $routeCollection = $loader->load('routing.yml');
+        $this->loader = new YamlFileLoader(new FileLocator([__DIR__ . '/../Fixtures/controller']));
```

Comment:
we still use array() vs [] for historical consistency (2.8 is still maintained for 1 year, after that we'll reconsider I hope)

Reply:
Ah, this is something PHPStorm did automatically on paste.

Reply:
@nicolas-grekas might be nice to have fabbot.io pick that up /cc @fabpot